frontend/device/kobo/sysfs_light.lua

```diff
@@ -0,0 +1,76 @@
+-- Interface to the SysFS interface of the Kobo Aura One Frontlight.
+-- This also supports the natural light, which consists of additional
+-- red and green light LEDs.
+
+local logger = require("logger")
+
+local KoboSysfsLight = {
+    frontlight_white = "/sys/class/backlight/lm3630a_led1b",
+    frontlight_red = "/sys/class/backlight/lm3630a_led1a",
+    frontlight_green = "/sys/class/backlight/lm3630a_ledb",
+    current_brightness = 0,
+    current_warmth = 0,
+    white_gain = 25,
+    red_gain = 24,
+    green_gain = 24,
+    white_offset = -25,
+    red_offset = 0,
+    green_offset = -65,
+}
+
+function KoboSysfsLight:setBrightness(brightness)
+    assert(brightness >= 0 and brightness <= 100,
+           "Wrong brightness value given!")
+    self:setNaturalBrightness(brightness, self.current_warmth)
+end
+
+function KoboSysfsLight:setWarmth(warmth)
+    assert(warmth >= 0 and warmth <= 100,
+           "Wrong warmth value given!")
+    self:setNaturalBrightness(self.current_brightness, warmth)
+end
+
+function KoboSysfsLight:setNaturalBrightness(brightness, warmth)
+    -- On Nickel, the values for white/red/greem are roughly linearly dependend
```

Comment:
With the very high quality of this PR, the typos in this comment stick out to me a bit. 😊 

(green, dependent)